### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by ghost**
In the file I have corrected a typo at the "hiding a column by changing style properties" instead of the "col === 0" I put "i === 0" because the variable "col" is not defined in the script. Thus, using col the code was not working. Also,, at the "Getting a text node from the table" there was a type at part b) (I am not sure of the change though.

### CODE REVIEW

File: files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md [hunk 1]
```diff
@@ -329,7 +329,7 @@ The basic steps to create the table in sample1.html are:
 
 This example introduces two new DOM attributes. First it uses the `childNodes` attribute to get the list of child nodes of myCell. The `childNodes` list includes all child nodes, regardless of what their name or type is. Like `getElementsByTagName()`, it returns a list of nodes.
 
-The differences are that (a) `getElementsByTagName()` only returns elements of the specified tag name; and (b) `getElementsByTagName()` returns descendants at any level, not just immediate children.
+The differences are that (a) `getElementsByTagName()` only returns elements of the specified tag name; and (b) `childNode()` returns descendants at any level, not just immediate children.
```
Comment (estelle) [inline files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md:332]: childNodes (plural) is a property, not a method. So, yes, the original is wrong, but i think we may want to write something like:

```suggestion
The differences are that (a) `getElementsByTagName()` only returns elements of the specified tag name; and (b) `childNodes` includes all descendants at any level, not just immediate children.
```

File: files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md [hunk 2]
```diff
@@ -375,7 +375,7 @@ Once you have the object in your JavaScript variable, you can set `style` proper
 
       for (let row = 0; row < 2; row++) {
         const myCurrentRow = document.createElement("tr");
-        for (let col = 0; col < 2; col++) {
+        for (let i = 0; col < 2; col++) {
```
Comment (estelle) [inline files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md:378]: this is defining `col` as 0, then immediately uses `col`, incrementing it. `i` is not used. The original was correct (though i didn't check the full page to see why we used "2")
```suggestion
        for (let col = 0; col < 2; col++) {
```

File: files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md (hunk 1 above)
Comment (faithchilo) [reply files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md:332]: Oh yes, I didn't understand the childNode but now I do. And I think your correction will make a better choice.

File: files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md (hunk 2 above)
Comment (faithchilo) [reply files/en-us/web/api/document_object_model/traversing_an_html_table_with_javascript_and_dom_interfaces/index.md:378]: I think with this part there is no change to make. It just stays the same like the original.

**Review by estelle — APPROVED** [review]
Thank you. Also, congratulations on your first merged MDN content PR. Welcome to he team!

**Pull request merged**